src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -281,5 +281,15 @@ public static class Diagnostics {
 			isEnabledByDefault: true,
 			description: "ICorsHeaderAppender should not be used, as it can introduce security vulnerabilities."
 		);
+
+		public static readonly DiagnosticDescriptor UseNamedArgsForInvocationWithLotsOfArgs = new DiagnosticDescriptor(
+			id: "D2L0032",

```

Comment:
Not a big deal since can find-and-replace easy, but I'm currently using 32 in my pragma stuff. Perhaps best practice to just use `D2LXXXX` or something until landing?

Reply:
That makes sense. I think the test that checks for unique ids will still pass because they are opaque strings.